gatsby-browser.js

```diff
@@ -1,8 +1,17 @@
 import React from 'react'
+import Cookie from 'js-cookie'
 import { ThemeProvider } from './src/context/ThemeContext'
 
 require('focus-visible')
 
 export const wrapRootElement = ({ element }) => (
   <ThemeProvider>{element}</ThemeProvider>
 )
+
+export const onClientEntry = () => {
+  window[`ga-disable-${process.env.GOOGLE_ANALYTICS_ID}`] = true
+
+  if (Cookie.get('accept_cookies')) {
```

Comment:
it's better to check it like that:

`if (Cookie.get('accept_cookies') === 'true') {`